Ginger/Ginger/GeneralWindows/HelpLayout/DrawArrow.cs

```diff
@@ -0,0 +1,64 @@
+﻿using System;
+using System.Windows;
+using System.Windows.Media;
+using System.Windows.Shapes;
+
+namespace Ginger.GeneralWindows.HelpLayout
+{
+    public sealed class DrawArrow 
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Add a 'private' constructor or the 'static' keyword to the class declaration.](https://app.codacy.com/app/Ginger-Automation/Ginger/pullRequest?prid=4555652)